Tries to install a desktop file in /usr/share/tmp instead of
/etc/xdg/autostart.

--- a/DisplayCAL/setup.py
+++ b/DisplayCAL/setup.py
@@ -709,9 +709,7 @@
             )
             data_files.append(
                 (
-                    autostart
-                    if os.geteuid() == 0 or prefix.startswith("/")
-                    else autostart_home,
+                    autostart,
                     [
                         os.path.join(
                             pydir,
